docs/API.md

```diff
@@ -1,681 +1,1085 @@
 }
 ```
 
- -  params:
-     -  id: the id of the test whose results we want to get.
-     -  language: the language of the user interface
- -  jsonrpc: « 2.0 »
- -  id: any kind of unique id allowing to match requests and responses
- -  method: the name of the called method
-
-**Response**:
-```
+Example response:
+```json
 {
-  "jsonrpc" : "2.0",
-  "id" : 140723510525000,
-  "result" : {
-    "params" : {
-.
-.
-TEST PARAMS (See *JSON-RPC Call 5*: start_domain_test)
-.
-.
-  },
-  "id": 8881,
-  "creation_time": "2014-08-05 12:00:13.401442",
-  "results": [
-    {
-      "module": 'DELEGATION',
-      "message": 'Messsage for DELEGATION/NAMES_MATCH in the language:fr'
-      "level": 'NOTICE',
+  "jsonrpc": "2.0",
+  "id": 6,
+  "result": {
+    "creation_time": "2016-11-15 11:53:13.965982",
+    "id": 25,
+    "hash_id": "c45a3f8256c4a155",
+    "params": {
+      "ds_info": [],
+      "client_version": "1.0.1",
+      "domain": "zonemaster.net",
+      "profile": "default_profile",
+      "ipv6": true,
+      "advanced": true,
+      "nameservers": [
+        {
+          "ns": "ns3.nic.se",
+          "ip": "2001:67c:124c:2007::45"
+        },
+        {
+          "ip": "192.93.0.4",
+          "ns": "ns2.nic.fr"
+        }
+      ],
+      "ipv4": true,
+      "client_id": "Zonemaster Dancer Frontend"
     },
-.
-.
-LIST OF TEST RESULTS
-.
-{
-  "ns": "ns1.nic.fr",
-  "module": "NAMESERVER",
-  "message": "Messsage for NAMESERVER/AXFR_FAILURE in the language:fr"
-  "level": "NOTICE",
-},
-.
-.
-LIST OF TEST RESULTS
-.
-.
-]
-}
+    "results": [
+      {
+        "module": "SYSTEM",
+        "message": "Using version v1.0.14 of the Zonemaster engine.\n",
+        "level": "INFO"
+      },
+      {
+        "message": "Configuration was read from DEFAULT CONFIGURATION\n",
+        "level": "INFO",
+        "module": "SYSTEM"
+      },
+      ...
+    ]
+  }
 }
 ```
 
- -  jsonrpc: « 2.0 »
- -  id: any kind of unique id allowing to match requests and responses
- -  result: Contains:
-     -  id: The id of the test whose results are returnes
-     -  creation\_time: The exact time the test was created
-     -  params: The parameters used to run this test (See *JSON-RPC Call
-        5*: start\_domain\_test)
-     -  results: A list of results.
+>
+> Note: The above example response was abbreviated for brevity to only include
+> the first two elments in each list. Omitted elements are denoted by a `...`
+> symbol.
+>
 
-#### Description of the results:
 
-The individual results are of the form
+#### `"params"`
 
-```
-{
-  "module": "DELEGATION",
-  "message": "Messsage for DELEGATION/NAMES_MATCH in the language:fr"
-  "level": "NOTICE",
-}
-```
+An object with the following properties:
 
-Or
+* `"id"`: A *test id*, required.
+* `"language"`: A string, required. Must be at least two characters long. The
+  two first characters are used to look up the *translation language* to be
+  used. If the lookup fails, the choice defaults to English.
```

Comment:
List supported language tags. If the lookup the messages can be broken since the even for English "translation" is needed.